azure/packages/test/end-to-end-tests/src/test/containerCreate.spec.ts

```diff
@@ -164,6 +172,7 @@ describe("Container create with feature flags", () => {
 	let client: AzureClient;
 	let schema: ContainerSchema;
 	let mockLogger: MockLogger;
+	const isEphemeral: boolean = true;
```

Comment:
qq..why are we setting isEphemeral to true here instead of using `isEphemeralContainer = process.env.IS_EPHEMERAL_CONTAINER === "true"`

Reply:
This specific part is only for the "Container create with feature flags" test case. The intent is that even if you run the durable container version of E2E there is still a small check to make sure that ephemeral containers can still be created. I figured this would be fitting to have for this specific test case, since it is supposed to validate the different feature flags.

Let me know if you agree with this logic, or if this is maybe something I should change/approve

Reply:
Sounds good to me!

Reply:
As an update to this, my recent change in response to Zach's comment makes the ephemeral flag be read from the environment in createAzureClient, so this part is no longer needed